service/src/main/java/org/cbioportal/genome_nexus/service/annotation/NotationConverter.java

```diff
@@ -154,6 +160,96 @@ else if (ref.length() > 1 || var.length() > 1) {
         return hgvs;
     }
 
+    @Nullable
+    public String genomicToEnsemblRestRegion(GenomicLocation genomicLocation)
```

Comment:
It would be nice to have some tests for non trivial methods like this one.

Reply:
added tests